src/modm/board/blue_pill_f103/board.hpp

```diff
@@ -24,13 +25,38 @@ namespace Board
 /// @{
 using namespace modm::literals;
 
-/// STM32F103 running at 72MHz generated from the external 8MHz crystal
 struct SystemClock
 {
-	static constexpr uint32_t Frequency = 72_MHz;
-	static constexpr uint32_t Ahb = Frequency;
-	static constexpr uint32_t Apb1 = Frequency / 2;
-	static constexpr uint32_t Apb2 = Frequency;
+	static constexpr uint32_t ExternalCrystalClock = 8_MHz;
+
+	static constexpr Rcc::PllFactors pllFactors{
+		.pllMul = 9,
+		.usbPrediv = Rcc::UsbPrescaler::Div1_5
+	};
+	static constexpr Rcc::AhbPrescaler Ahb_pre = Rcc::AhbPrescaler::Div1;
+	static constexpr Rcc::Apb1Prescaler Apb1_pre = Rcc::Apb1Prescaler::Div2;
+	static constexpr Rcc::Apb2Prescaler Apb2_pre = Rcc::Apb2Prescaler::Div1;
```

Comment:
One can now choose arbitrary pll values and prescalers here.

But there's a flaw: When setting anything than `Div1` or `Div2` to `Rcc::AhbPrescaler`, f103-blue-bill refuses to work. f411-disco keeps working but the Timer frequencies no more correspond to my determined values.

I've matched the varrious intermediate clocks with what CubeMX thinks they should be. But anyways i couldn't determine a logic error in my changes.

Propably, another side effect enters the show when going beyond Div1, Div2 with this Ahb-prescaler. Since the Ahb-prescaler is almost always set to Div1, lets be happy with this for this PR.